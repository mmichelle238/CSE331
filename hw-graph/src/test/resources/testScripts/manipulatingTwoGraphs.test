# Tests graph creation/viewing with a
# simple graph ('A') with three nodes.
# And the simultaneous creation of another graph
# ('B') does not result in incorrect behavior.

CreateGraph graph1
AddNode graph1 A
AddNode graph1 B
CreateGraph graph2
AddNode graph2 A
AddNode graph2 B
AddEdge graph1 A B 5
AddEdge graph2 A B 5
AddEdge graph1 B A 3
AddEdge graph2 B A 4
AddEdge graph1 A B 7
AddEdge graph2 A B 7