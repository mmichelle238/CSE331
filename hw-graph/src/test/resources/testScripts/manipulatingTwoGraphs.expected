# Tests graph creation/viewing with a
# simple graph ('A') with three nodes.
# And the simultaneous creation of another graph
# ('B') does not result in incorrect behavior.

created graph graph1
added node A to graph1
added node B to graph1
created graph graph2
added node A to graph2
added node B to graph2
added edge 5 from A to B in graph1
added edge 5 from A to B in graph2
added edge 3 from B to A in graph1
added edge 4 from B to A in graph2
added edge 7 from A to B in graph1
added edge 7 from A to B in graph2